lib/tmuxinator/project.rb

```diff
@@ -208,7 +217,12 @@ def tmux_options
     end
 
     def base_index
-      get_base_index.to_i
+      base = 0
+      if in_current_session
+        # Get the last window index + 1
+        base = 1 + `tmux list-windows | awk '{print $1}'`.split.last.to_i
```

Comment:
Tell tmux to only print the window indices:

```suggestion
        base = 1 + `tmux list-windows -F '#I'`.split.last.to_i
```

The output is like so:

```
1
2
3
```